src/AllWorkspaceTestsRunner.java

```diff
@@ -1,76 +0,0 @@
-import java.io.File;
```

Comment:
Unused, you can do the same thing with patterns in ant anyway (and presumably Gradle, etc.)